tests/python/frontend/tflite/test_forward.py

```diff
@@ -1850,6 +1878,16 @@ def _test_tan(data):
     return _test_unary_elemwise(math_ops.tan, data)
 
 
+#######################################################################
+# Sqrt
+# ----
+
+
+def _test_sqrt(data):
+    """One iteration of sqrt"""
+    return _test_unary_elemwise(math_ops.sqrt, data)
+
+
```

Comment:
This block above seems unrelated to this PR, right?

Reply:
Yes I will remove them.